apps/ui/src/components/EditorVote.vue

```diff
@@ -2,6 +2,7 @@
 import Draggable from 'vuedraggable';
 import { BASIC_CHOICES } from '@/helpers/constants';
 import { Draft, VoteType } from '@/types';
+import ICDrag from '~icons/c/drag';
```

Comment:
You don't need this import, they are globally available already

Reply:
They were import in Home.vue and Overview.vue 🙈

Reply:
Because they are used in a list in script setup